purchase_packaging_report/report/purchase_quotation_templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchasequotation_document"
+        inherit_id="purchase.report_purchasequotation_document"
+    >
+        <xpath expr="//td/span[@t-field='order_line.product_qty']/.." position="inside">
+            <div
+                t-if="order_line.product_packaging_id"
+                groups="product.group_stock_packaging"
+                class="text-muted d-inline-block"
+            >
+                <span>(</span>
+                <span t-field="order_line.product_packaging_qty" />
+                <span class="mx-0">x</span>
+                <span t-field="order_line.product_packaging_id.name" />
+                <span>)</span>
```

Comment:
Same as above:

```suggestion
                (<span t-field="order_line.product_packaging_qty" />
                <span class="mx-0">✕</span>
                <span t-field="order_line.product_packaging_id.name" />)
```

Reply:
Changed to maintain the same appearance as sale_packaging_report